panoramix/assets/visualizations/pivot_table.css

```diff
@@ -2,7 +2,12 @@
     overflow: auto !important;
 }
 
-.widget.pivot_table td,th {
-    padding: 1px 5px;
-    font-size: small;
+.table tr>th {
```

Comment:
I'm not sure how else to compete with the very high selectivity css in the bootstrap/dataTables css...